cluster/images/etcd/migrate/migrate_server.go

```diff
@@ -102,18 +102,12 @@ func (r *EtcdMigrateServer) Stop() error {
 	if err != nil {
 		return fmt.Errorf("error sending SIGINT to etcd for graceful shutdown: %v", err)
 	}
-	gracefulWait := time.Minute * 2
```

Comment:
I'd keep the name and make it a constant.

Reply:
Sure, makes sense. Changed it to const. Thank you.